library/core/src/ptr/const_ptr.rs

```diff
@@ -723,7 +723,7 @@ impl<T: ?Sized> *const T {
     ///
     /// See [`ptr::read`] for safety concerns and examples.
     ///
-    /// [`ptr::read`]: ./ptr/fn.read.html
+    /// [`ptr::read`]: read()
```

Comment:
Oh wow, this is confusing: I'd expect it to link to the `read` in this module, which is the item being documented. But since this is in an `impl`, it doesn't find the current item, instead it sees the `read` from the glob import at the top.

It _works_, but do you mind changing it to something less confusing?
```suggestion
    /// [`ptr::read`]: crate::ptr::read()
```